test/test_css_parser_loading.rb

```diff
@@ -197,10 +197,13 @@ def test_suppressing_circular_reference_exceptions
   def test_toggling_not_found_exceptions
     cp_with_exceptions = Parser.new(:io_exceptions => true)
 
-    assert_raises RemoteFileError do
+    err = assert_raises RemoteFileError do
       cp_with_exceptions.load_uri!("#{@uri_base}/no-exist.xyz")
     end
 
+    uri_regex = Regexp.new(Regexp.escape("#{@uri_base}/no-exist.xyz"))
+    assert_match uri_regex, err.message
```

Comment:
this might be simpler and produce a more readable error
`err.message.must_include "#{@uri_base}/no-exist.xyz"`